.github/CODEOWNERS

```diff
@@ -1,6 +1,4 @@
-# By default, all files require a review by at least one member of the Azure Linux developers team.
 # See teams here: https://github.com/orgs/microsoft/teams?query=mariner
-* @microsoft/cbl-mariner-devs
```

Comment:
Do you know who will be allowed to approve a PR for files not captured in this file after this change? I think at the moment this may be everyone. We can change the [moderation options](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-pull-request-reviews-in-your-repository) for the repo to limit the set of users who can approve PRs to anyone, who's been explicitly granted read access or higher (assuming that's not already enabled on the org level), but I'm not sure if this is still not broad enough. Do we want give approval permissions to @microsoft/cbl-mariner-guest-devs for instance?